compiler/codegen/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![deny(warnings)]
```

Comment:
As a rule with the amount of churn in the codegen/compiler crates, I'd prefer to not have this set - it makes it far far too annoying to iterate during development

Reply:
I can turn it back off, I turned it on to fix the old warnings, so I could see the new warnings I made with the updates easier.